spec/factories/payment_responses.rb

```diff
@@ -0,0 +1,86 @@
+FactoryBot.define do
+  sequence :adyen_payments_response do
+    OpenStruct.new(
+      response: {
+        "additionalData" => {
+            "recurringProcessingModel" => "UnscheduledCardOnFile"
```

Comment:
Style/TrailingCommaInHashLiteral: Put a comma after the last item of a multiline hash.